shell.nix

```diff
@@ -107,6 +116,7 @@ haskell.project.shellFor {
 
   shellHook = ''
     ${pre-commit-check.shellHook}
```

Comment:
Is this in the new devshell?

Reply:
The pre-commit-check is in the new shell